Makefile

```diff
@@ -22,6 +22,8 @@ GOOSES   ?= "linux windows" # To override at the cli do: GOOSES="\"darwin bsd\""
 GOARCHES ?= "amd64 arm64" # To override at the cli do: GOARCHES="\"ppc64 mips\""
 ltsc2019  = "10.0.17763.4010"
 ltsc2022  = "10.0.20348.643"
+ltsc2022-KB5035857   = "10.0.26100.1"
+win2025 = "10.0.26100.1"
```

Comment:
How is "10.0.26100.1" got determined?

Reply:
This is the platform of l1vh: PS C:\k> systeminfo | findstr /B /C:"OS Name" /C:"OS Version"
OS Name:                       Microsoft Windows Server 2025 Datacenter
OS Version:                    10.0.26100 N/A Build 26100